lib/builders/member/memberValueBuilder.ts

```diff
@@ -0,0 +1,39 @@
+﻿import {MemberBuilder} from "./memberBuilder";
```

Comment:
The fileNames for memberValueBuilder and memberVariantBuilder seems to be switched.